src/main/cpp/subsystems/Chassis.cpp

```diff
@@ -1,6 +1,8 @@
 #include "subsystems/Chassis.h"
 #include "RobotMap.h"
 #include "commands/TankDrive.h"
+#include <networktables/NetworkTable.h>
```

Comment:
`#include <networktables/NetworkTable.h>` is in the hpp file. Not needed here.

I'll fix this on the rebase.